wikimedia/wikipedia-ios
# Image Recommendations - Add feature announcement modal

**Phabricator:** https://phabricator.wikimedia.org/T358894

### Notes
Adds a feature announcement component and presents via a half sheet modal. This one may have major design changes, so marking as Draft.

### Test Steps
1. Launch on app, be sure you are logged into an unblocked account on a wiki with 50+ edits.
2. Confirm feature announcement modal displays once per install upon landing on Explore feed, if the suggested edits card is visible.

Note: Sometimes another modal appears before this one has a chance to present. Dismissing the modal, then leaving the tab and returning to the Explore feed will trigger another attempt to present this one.

### Screenshots/Videos
